cucumber/authenticators_gcp/features/authn_gcf_token_errors.feature

```diff
@@ -0,0 +1,93 @@
+Feature: GCP Authenticator - Test Token Error Handling
+
+  In this feature we test authentication using malformed tokens.
+  Will verify a failure of the authentication request in such a case
+  and log of the relevant error.
+
+  Background:
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-gcp
+      body:
+      - !webservice
+
+      - !group apps
+
+      - !permit
+        role: !group apps
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I have host "test-app"
+    And I grant group "conjur/authn-gcp/apps" to host "test-app"
+
+  Scenario: Token with service-account-id claim that does not match annotation is denied
+    And I set "authn-gcp/service-account-id" annotation with value: "unknown-service-account-id" to host "test-app"
```

Comment:
Relevant to the whole PR:
Remove annoations before setting a new one

Reply:
How do I remove annotation?

Reply:
see previous comments i mentioned a step there